install/mlops/comet/install.yml

```
@@ -0,0 +1,14 @@
+id: comet-mlops
+name: Comet
+title: 
```

This also needs a title
```suggestion
title: Comet
```